Task1/script.js

```diff
@@ -0,0 +1,39 @@
+const th = table.querySelectorAll('th');
+const tbody = table.querySelector('tbody');
```

Comment:
how you get table? you need to use `querySelector` for it and don't get by id from  `document` because sometime you can get undefined if `dom` doesn't ready